news/_posts/2017-02-11-ev3dev-jessie-2017-02-11-release.md

```diff
@@ -0,0 +1,58 @@
+---
+author: "@dlech"
+title: "Announcing ev3dev-jessie-2017-02-11 Release"
+---
+
+This is a stable maintenance release. Although we did manage to sneak a few
```

Comment:
This is an odd usage of "although"; there's no conclusion made based off the condition. "However, ...", "Despite that," or "That being said" seems more natural.